packages/php/blueprint/src/Importers/ImportActivateTheme.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Importers;
+
+use Automattic\WooCommerce\Blueprint\StepProcessor;
+use Automattic\WooCommerce\Blueprint\StepProcessorResult;
+use Automattic\WooCommerce\Blueprint\Steps\ActivatePlugin;
+use Automattic\WooCommerce\Blueprint\Steps\ActivateTheme;
+use Automattic\WooCommerce\Blueprint\UsePluginHelpers;
+use Automattic\WooCommerce\Blueprint\UseWPFunctions;
+
+class ImportActivateTheme implements StepProcessor {
+	use UsePluginHelpers;
+	use UseWPFunctions;
+
+	public function process( $schema ): StepProcessorResult {
+		$result = StepProcessorResult::success( 'DeactivatePlugins' );
```

Comment:
Is `DeactivatePlugins` intended?

Reply:
Updated in [Use the correct step names](https://github.com/woocommerce/woocommerce/pull/49763/commits/998c76341c3cb81d94781d5e5e0cb047aa30a86b)